easybuild/easyconfigs/g/GROMACS/GROMACS-2021.5-foss-2021b-CUDA-11.4.1.eb

```diff
@@ -9,13 +9,15 @@
 # * Kenneth Hoste <kenneth.hoste@ugent.be>
 # * Adam Huffman <adam.huffman@crick.ac.uk>
 # * Ake Sandgren <ake.sandgren@hpc2n.umu.se>
-# * J. Sassmannshausen <Crick HPC team>
 # * Dugan Witherick <dugan.witherick@warwick.ac.uk>
 # * Christoph Siegert <christoph.siegert@uni-leipzig.de>
+# * J. Sassmannshausen <j.sassmannshausen@imperial.ac.uk>
 # License::   MIT/GPL
 
 name = 'GROMACS'
 version = '2021.5'
+_plumedver = '2.8.0'
+versionsuffix = '-PLUMED-%s' % _plumedver
 versionsuffix = '-CUDA-%(cudaver)s'
```

Comment:
```suggestion
_cuda_suffix = '-CUDA-%(cudaver)s'
versionsuffix = _cuda_suffix + '-PLUMED-%s' % _plumedver
```

Reply:
Thanks. That slipped my attention. That might explain the wrong name of the EC file. I changed that.

Reply:
Can you also remove the changes to `GROMACS-2021.5-foss-2021b-CUDA-11.4.1.eb` from this PR?

Reply:
Done (hopefully). I was a bit confused as I had locally the 'correct' filename, i.e. the one I wanted to have. I hope that is now all settled.